src/views/home/home.view.tsx

```diff
@@ -34,6 +35,10 @@ const Home = (): JSX.Element => {
     setFormData(undefined);
   };
 
+  const onChangeNetwork = (chain: Chain) => {
+    void changeNetwork(chain);
```

Comment:
We should catch this error and report it when `isMetamaskUserRejectedRequestError` evaluates as `false`. And we should do it using `callIfMounted`:

```typescript
changeNetwork(chain).catch((error) => {
  callIfMounted(() => {
    if (isMetamaskUserRejectedRequestError(error) === false) {
      notifyError(error);
    }
  });
});
```

Reply:
I don't think so because in this case the action is `Add to MetaMask` and if the user cancels it, I think that the experience has stopped and the user knows it

Reply:
Sorry I was thinking in the other flow